src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
Ensure that the `Person` component is correctly implemented to handle the `person` prop. Specifically, check if it can handle cases where some properties might be missing, such as `age` for `olya`.